### PR TITLE
Concepts: some more typos spotted when reading the docs.

### PR DESCRIPTION
**Pull request by didier-durand**
Hi there,
As per title.
Didier
Signed-off-by: Didier Durand <durand.didier@gmail.com>

### CODE REVIEW

File: .project [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (floreks) [inline .project:1]: What is this file? Is it needed?

**Comment by didier-durand** [comment]
@floreks: my bad! It is a file of my IDE. I removed it now. Didier

**Pull request merged**